### PR TITLE
Reviewer edits

### PR DESCRIPTION
**Pull request by ejm714**
See reviewer comments [here](https://github.com/scipy-conference/scipy_proceedings/pull/936)

### CODE REVIEW

**Comment by ejm714** [comment]
@klwetstone I took a pass at responding to the reviewer's comments in this PR. can you take a look and see if it makes sense and wordsmith accordingly?

**Review by klwetstone — APPROVED** [review]
@ejm714 This looks good to me! I pushed a few small copy edits.

My main remaining question is about how we address samples that were dropped from train/test because we could not get valid satellite imagery. We mentioned briefly how many were dropped from the test set in a new footnote, and I think it would be helpful to also include somewhere how many were dropped from the training set.

Ideally, I think we would include this in the "Satellite feature engineering decisions" sections similar to what we've done with our data filters.

**Comment by klwetstone** [comment]
> My main remaining question is about how we address samples that were dropped from train/test because we could not get valid satellite imagery. We mentioned briefly how many were dropped from the test set in a new footnote, and I think it would be helpful to also include somewhere how many were dropped from the training set.
> 
> Ideally, I think we would include this in the "Satellite feature engineering decisions" sections similar to what we've done with our data filters.

@ejm714 I can take this on today

**Comment by ejm714** [comment]
Going to merge this in as is and we can add the train note separately if desired
